app/views/layouts/emailTemplates.volt

```diff
@@ -1,50 +1,49 @@
 <html>
 <head></head>
 <body style="background-color: #E4E4E4;padding: 20px; margin: 0; min-width: 640px;">
-	<table border="0" cellspacing="0" width="530" style="color:#262626;background-color:#fff;
-		padding:27px 30px 20px 30px;margin:auto; border:1px solid #e1e1e1;">
-		<tbody>
-			<!-- header -->
-			<tr style="background:#D44413">
-				<td style="padding-left:20px">
-					<a target="_blank" style="text-decoration:none;color:inherit;font-family:'HelveticaNeue','Helvetica Neue',Helvetica,Arial,sans-serif;font-weight:normal;">
-						<h1 style="color:#fff">Vökuró</h1>
-					</a>
-				</td>
-			</tr>
-		</tbody>
+<table border="0" cellspacing="0" width="530" style="color:#262626;background-color:#fff;padding:27px 30px 20px 30px;margin:auto;border:1px solid #e1e1e1;">
+    <tbody>
+    <!-- header -->
+    <tr style="background:#D44413">
+        <td style="padding-left:20px">
+            <a target="_blank" style="text-decoration:none;color:inherit;font-family:'HelveticaNeue','Helvetica Neue',Helvetica,Arial,sans-serif;font-weight:normal;">
+                <h1 style="color:#fff">Vökuró</h1>
+            </a>
+        </td>
+    </tr>
+    </tbody>
 
-		{{ content() }}
+    {{ content() }}
 
-		<!--footer-->
-		<tbody>
-			<tr>
-				<td align="right" style="padding:25px 0  0 0;">
-					<table border="0" cellspacing="0" cellpadding="0" style="padding-bottom:9px;" align="right">
-						<tbody>
-							<tr style="border-bottom:1px solid #999999;">
-								<td width="24" style="padding:0 7px 0 0;">
-									<a href="http://www.facebook.com/pages/Phalcon/134230726685897" style="border-width:0;">
-										<img src="http://{{ publicUrl }}/img/facebook.gif" width="24" height="24" alt="Facebook Image">
-									</a>
-								</td>
-								<td width="24">
-									<a href="http://twitter.com/phalconphp" style="border-width:0;">
-										<img src="http://{{ publicUrl }}/img/twitter.gif" width="24" height="24" alt="Twitter Image">
-									</a>
-								</td>
-							</tr>
-							<tr style="height:1px; background:#000;padding-top:15px">
-								<td></td>
-								<td></td>
-								<td></td>
-							</tr>
-						</tbody>
-					</table>
-				</td>
-			</tr>
-		</tbody>
-	</table>
+    <!--footer-->
+    <tbody>
+    <tr>
+        <td align="right" style="padding:25px 0  0 0;">
+            <table border="0" cellspacing="0" cellpadding="0" style="padding-bottom:9px;" align="right">
+                <tbody>
+                <tr style="border-bottom:1px solid #999999;">
+                    <td width="24" style="padding:0 7px 0 0;">
+                        <a href="http://www.facebook.com/pages/Phalcon/134230726685897" style="border-width:0;">
+                            <img src="http://{{ publicUrl }}/img/facebook.gif" width="24" height="24" alt="Facebook Image">
```

Comment:
Can we add http(s) detection? If you are running on https this won't work.

Reply:
Yes and let us use the phalcon.link URL shorteners

Reply:
3fdd167

Reply:
d8b5d2f